### PR TITLE
Modifying the Form - Validation demo 23_1

### PR DESCRIPTION
**Pull request by 16adianay**

### CODE REVIEW

File: JSDemos/Demos/Form/Validation/Angular/app/app.component.ts [hunk 1]
```diff
@@ -38,11 +39,41 @@ const sendRequest = function (value) {
 export class AppComponent {
   @ViewChild(DxFormComponent, { static: false }) form:DxFormComponent;
 
-  password = '';
-
   passwordOptions: any = {
```
Comment (Zedwag) [inline JSDemos/Demos/Form/Validation/Angular/app/app.component.ts:42]: u forgot to pass these two objects to editorOptions in html

File: JSDemos/Demos/Form/Validation/Angular/app/app.component.ts [hunk 2]
```diff
@@ -75,6 +106,14 @@ export class AppComponent {
     return true;
   }
 
+  changePasswordMode = (name) => {
+    let editor = this.form.instance.getEditor(name);
```
Comment (Zedwag) [inline JSDemos/Demos/Form/Validation/Angular/app/app.component.ts:110]: and name to confirm passport editor :)

**Review by Zedwag — COMMENTED** [review]
lgtm

File: JSDemos/Demos/Form/Validation/Angular/app/app.component.ts (hunk 2 above)
Comment (16adianay) [reply JSDemos/Demos/Form/Validation/Angular/app/app.component.ts:110]: thanks!)

**Review by Zedwag — APPROVED** [review]

**Pull request merged**